src/components/ConfiguredToast.jsx

```diff
@@ -0,0 +1,18 @@
+import { ToastContainer } from 'react-toastify';
+
+export function ConfiguredToast(){
+  return (
```

Comment:
Moving the configured ToastContainer into this component works because, at the end of the day, a component is just a function that returns some JSX. In the case of this component, in App, ConfiguredToast is just returning an already configured ToastContainer, which in turn is returning its own JSX to do its magic.